doc/ref_cert/RC2/chapters/chapter03.md

```diff
@@ -154,12 +154,21 @@ weaknesses in Kubernetes clusters and
 whether Kubernetes is deployed securely by running the checks documented in the
 [CIS Kubernetes Benchmark](https://www.cisecurity.org/benchmark/kubernetes/).
 
-At the time of writing, no clear security rules are defined as mandatory
-(e.g. sec.std.001: The Cloud Operator **should** comply with Center for
-Internet Security CIS Controls) which would have asked for an update of the
-default
-[Functest Kubernetes Security](https://git.opnfv.org/functest-kubernetes/tree/docker/security/testcases.yaml?h=stable%2Fkali)
-behavior (all failures and warnings are only printed).
+The following
+[vulnerability categories](https://github.com/aquasecurity/kube-hunter/blob/v0.3.1/kube_hunter/core/events/types.py)
+tagged as high by
+[kube-hunter](https://github.com/aquasecurity/kube-hunter) lead to a test case
+failure:
+- RemoteCodeExec
+- IdentityTheft
+- PrivilegeEscalation
```

Comment:
What's the impact of this - i.e. as a reader, why is this useful, what do I need to do?  Should we know why?  Is this just for info and we can ignore the failures?

Reply:
It's about the test case description here and then I would consider that the test case criteria must be explicit for the end users. It tried to describe that the test case fails if high vulnerabilities (mostly CVE) are detected. 
It's worth mentioning here that the minor and medium vulnerabilities are printed but the test case is considered as PASS.

Reply:
OK - could you add some context to the doc please?  As written, if I read that, I wonder what I need to do to make the test case pass, or whether I can ignore it.